### PR TITLE
murdock: Add basic prometheus instrumentation

### PR DESCRIPTION
**Pull request by bergzand**
This adds python and HTTP statistics to be scraped by a prometheus (or other openmetrics collector) at the `/metrics` endpoint.

### CODE REVIEW

**Comment by bergzand** [comment]
This can be extended later with murdock-specific metrics such as job and build counts

**Comment by bergzand** [comment]
I can reproduce the build failures locally. I also get them when running the tests on current `main`

**Comment by bergzand** [comment]
Rebased to include #11

File: murdock/main.py [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 from jinja2 import FileSystemLoader, Environment
 
+
```
Comment (aabadie) [inline murdock/main.py:25]: nit: this seems unrelated

Comment (bergzand) [reply murdock/main.py]: Correct, removed and pushed again

**Comment by bergzand** [comment]
And rebased

File: murdock/main.py [hunk 2]
```diff
@@ -82,6 +81,12 @@
 )
 
 
+@app.on_event("startup")
+async def startup():
+    murdock.instrumentator.instrument(app).expose(app)
```
Comment (aabadie) [inline murdock/main.py:86]: Why not simply move this to the murdock.init method ?

Comment (bergzand) [reply murdock/main.py:86]: And pass the `app` as argument to the `murdock.init` method?

Comment (bergzand) [reply murdock/main.py:86]: And pass the app as argument to the murdock.init method?

Comment (aabadie) [reply murdock/main.py:86]: ah you are right, this has to be here...

**Review by aabadie — APPROVED** [review]
